eZ/Publish/API/Repository/Values/User/RoleDraft.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * File containing the eZ\Publish\API\Repository\Values\User\RoleDraft class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ *
+ * @version //autogentag//
+ */
+namespace eZ\Publish\API\Repository\Values\User;
+
+/**
+ * This class represents a draft of a role.
+ *
+ * @property-read mixed $id the internal id of the role
+ * @property-read string $identifier the identifier of the role
+ *
+ * @property-read array $policies an array of the policies {@link \eZ\Publish\API\Repository\Values\User\Policy} of the role.
```

Comment:
The same already exist in the extended class, it should not be needed to redefine them.